npm/svelte/src/mount.ts

```diff
@@ -76,7 +76,7 @@ export function mount<T extends SvelteComponent> (
     // by waiting, we are delaying test execution for the next tick of event loop
     // and letting hooks and component lifecycle methods to execute mount
     return cy.wait(0, { log: false }).then(() => {
-      if (options.log) {
+      if (options.log !== false) {
```

Comment:
Nice, all our adapters do this - this could be part of GA, the standard API that adapters must comply with. Part of that could be logging the component name, and we could enforce it via an interface or something.